extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -112,34 +107,32 @@ class AmpBridPlayer extends AMP.BaseElement {
     }
   }
 
-  /** @private */
-  buildImagePlaceholder_() {
-    const imgPlaceholder = new Image();
+  /** @override */
+  createPlaceholderCallback() {
+    const placeholder = this.win.document.createElement('amp-img');
     const partnerID = this.partnerID_;
     const feedID = this.feedID_;
 
-    setStyles(imgPlaceholder, {
-      'object-fit': 'cover',
-      // Hiding the placeholder initially to give the browser time to fix
-      // the object-fit: cover.
-      'visibility': 'hidden',
-    });
-
-    imgPlaceholder.src = 'https://cdn.brid.tv/live/partners/' + encodeURIComponent(partnerID) + '/snapshot/' + encodeURIComponent(feedID) + '.jpg';
-    imgPlaceholder.setAttribute('placeholder', '');
-    imgPlaceholder.setAttribute('referrerpolicy', 'origin');
-
-    this.applyFillContent(imgPlaceholder);
-    this.element.appendChild(imgPlaceholder);
-
-    this.loadPromise(imgPlaceholder).catch(() => {
-      imgPlaceholder.src = 'https://cdn.brid.tv/live/default/defaultSnapshot.png';
-      return this.loadPromise(imgPlaceholder);
-    }).then(() => {
-      setStyles(imgPlaceholder, {
-        'visibility': '',
-      });
-    });
+    const placeholderFallback = this.win.document.createElement('amp-img');
+    placeholderFallback.setAttribute('src',
+        'https://cdn.brid.tv/live/default/defaultSnapshot.png');
+    placeholderFallback.setAttribute('referrerpolicy', 'origin');
+    placeholderFallback.setAttribute('layout', 'fill');
+    placeholderFallback.setAttribute('fallback', '');
+    placeholder.appendChild(placeholderFallback);
+
+    // TODO(#5321): Investigate if this is actually the correct URL format for generated
+    // dynamic image.
+    placeholder.setAttribute('src',
```

Comment:
Yes if alt is supported on the element we should propagate it. Will review to add it where supported